src/router/router.js

```diff
@@ -32,10 +35,16 @@ const routes = [
     },
     {
         path: '/product_create',
-        name: 'Ceate product',
+        name: 'CeateProduct',
```

Comment:
Опечатка в слове create
```suggestion
        name: 'CreateProduct',
```

Reply:
коммит 10079f88